src/style.css

```diff
@@ -1,3 +1,79 @@
+html {
+  scroll-behavior: smooth;
+  font-size: 12px;
+  margin: 0;
+  padding: 0;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  max-height: 60px;
```

Comment:
The `max-height: 60px;` property in the `body` selector might be too restrictive and could cause layout issues if the content exceeds this height. Consider removing it or adjusting it according to the design requirements.